src/plugins/platforms/freerdp/qfreerdppeer.cpp

```diff
@@ -1415,10 +1415,10 @@ void QFreeRdpPeer::handleVirtualKeycode(quint32 flags, quint32 vk_code) {
 		vk_code |= KBDEXT;
 
 	// get scan code
-#if FREERDP_VERSION_MAJOR < 3
-	quint32 scancode = GetKeycodeFromVirtualKeyCode(vk_code, KEYCODE_TYPE_EVDEV);
-#else
+#if FREERDP_VERSION_MAJOR == 3 && FREERDP_VERSION_MINOR >= 5 || FREERDP_VERSION_MAJOR > 3
```

Comment:
Same here I think `WINPR_KEYCODE_TYPE_XKB` has been introduced quite early in the 3.0 cycle